x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/aggs/frequentitemsets/FrequentItemSetsAggregationBuilder.java

```diff
@@ -181,6 +181,11 @@ public boolean supportsSampling() {
         return true;
     }
 
+    @Override
+    public boolean supportsParallelCollection() {
```

Comment:
I assume that this agg didn't support parallel collection also before this change?

Reply:
Indeed, the postcollection method is accessing doc values created on the coordinating thread.